docs/main/quickstart-guide/installation.md

```diff
@@ -43,6 +43,21 @@ At a high-level, OpenEBS requires:
 
 ## Installation via Helm
 
+:::warning
+**Deprecation Notice: Helm Chart Registry Migration**
+
+The Helm chart registry at "https://openebs.github.io/charts" is deprecated and will be moved to a new location soon.
```

Comment:
This and the below 2 lines give conflicting messages

Reply:
@avishnu Then, can we remove the "**Deprecation Notice** Warning" & keep only the "**Important** panel" (i.e. in line 70)

Reply:
I think the text message needs to be modified. Something like:

The Helm chart registry at "https://openebs.github.io/charts" has now been deprecated as those charts are used to install legacy OpenEBS (v3.10 and below) releases. This registry will be migrated to a different registry location "https://openebs-archive.github.io/charts" by Oct 30, 2024.

The Helm charts for the latest OpenEBS (v4.0 and above) are hosted in a new registry location "https://openebs.github.io/openebs". To ensure seamless access to OpenEBS Helm charts, update your configurations to use the new registry URL.